clang/lib/CIR/CodeGen/CIRGenOpenACCClause.h

```diff
@@ -36,6 +37,76 @@ template <typename ToTest> constexpr bool isCombinedType = false;
 template <typename T>
 constexpr bool isCombinedType<CombinedConstructClauseInfo<T>> = true;
 
+namespace {
+struct DataOperandInfo {
+  mlir::Location beginLoc;
+  mlir::Value varValue;
+  llvm::StringRef name;
+  mlir::ValueRange bounds;
+
+  DataOperandInfo(mlir::Location beginLoc, mlir::Value varValue,
```

Comment:
Is an explicit constructor necessary? Do you intend to add something other than 1:1 member initialization later?

Reply:
There is some oddity I never looked into about the `mlir::ValueRange` which resulted in a warning about a dangling reference.  Instead, I've decided to remove the `bounds` from this, and I'll deal with it in a future patch, which means I can remove this.